src/PayU/Alu/Response.php

```diff
@@ -355,6 +348,25 @@ public function setRrn($rrn)
         $this->rrn = $rrn;
     }
 
+    /**
```

Comment:
My intention was to bring consistency in Response.php file.
I've changed the dockblock for all properties to be consistent across all project